packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html

```diff
@@ -231,3 +231,53 @@ <h4 class="pt-3">
     </form>
     <p data-translate="telephony_line_softphone_save_theme"></p>
 </oui-collapsible>
+<oui-collapsible
+    class="custom-collapsible-no-click mt-3"
+    data-expanded="true"
+    data-heading="{{ 'telephony_line_softphone_logo_title' | translate }}"
+>
+    <form name="form" data-ng-submit=":: $ctrl.applyTheme()">
+        <div class="row">
+            <oui-field
+                data-ng-class="{
+                                    'col-md-10': $ctrl.logoFilename!=='',
+                                    'col-md-12': $ctrl.logoFilename===''
+                                }"
+                data-label="{{ 'telephony_line_softphone_your_logo' | translate }}"
+                data-help-text="{{ $ctrl.acceptedFormatsDesc + ' (< '+ $ctrl.maxSizeLogoFile/1000 + ' KB)' }}"
+            >
+                <oui-file
+                    data-accept="{{ $ctrl.acceptedFormats }}"
+                    data-maxsize="$ctrl.maxSizeLogoFile"
+                    data-on-select="$ctrl.updateFilesList()"
+                    data-droparea
+                    data-model="$ctrl.fileModel"
+                >
+                </oui-file>
+            </oui-field>
+            <img
+                class="col-md-2"
+                data-ng-hide="$ctrl.logoFilename===''"
```

Comment:
if `null` or `undefined` ?

Reply:
default value returned by api is '' (void string) when no logo

Reply:
ok so can you just replace `ngHide` by `ngIf`

Reply:
it will not polute the dom indeed thks!